src/panels/config/users/ha-user-picker.js

```diff
@@ -0,0 +1,109 @@
+import '@polymer/paper-fab/paper-fab.js';
+import '@polymer/paper-item/paper-item.js';
+import '@polymer/paper-card/paper-card.js';
+import '@polymer/paper-item/paper-item-body.js';
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { PolymerElement } from '@polymer/polymer/polymer-element.js';
+
+import '../../../layouts/hass-subpage.js';
+
+import LocalizeMixin from '../../../mixins/localize-mixin.js';
+import NavigateMixin from '../../../mixins/navigate-mixin.js';
+
+
+let registeredDialog = false;
+
+/*
+ * @appliesMixin LocalizeMixin
+ * @appliesMixin NavigateMixin
+ */
+class HaUserPicker extends NavigateMixin(LocalizeMixin(PolymerElement)) {
+  static get template() {
+    return html`
+  <style include="iron-flex ha-style">
+    paper-fab {
+      position: fixed;
+      bottom: 16px;
+      right: 16px;
+      z-index: 1;
+    }
+    paper-card {
+      display: block;
+      max-width: 600px;
+      margin: 16px auto;
+    }
+    a {
+      color: var(--primary-text-color);
+    }
+  </style>
+
+  <hass-subpage header="[[localize('ui.panel.config.users.picker.title')]]">
+    <paper-card>
+      <template is="dom-repeat" items="[[users]]" as="user">
+        <a href='[[_computeUrl(user)]]'>
+          <paper-item>
+            <paper-item-body two-line>
+              <div>[[_withDefault(user.name, 'Unnamed User')]]</div>
+              <div secondary="">[[user.id]]</div>
+            </paper-item-body>
+            <iron-icon icon="hass:chevron-right"></iron-icon>
+          </paper-item>
+        </a>
+      </template>
+    </paper-card>
+
+    <paper-fab
+      is-wide$="[[isWide]]"
```

Comment:
doesn't look like an attribute for the fab

Reply:
Ah yes, that's a CSS styling artifact from copying it from automation editor. I will add the CSS back. It moves the fab a bit further away from the edge on wide screens as per the material spec.